zeroex/ordervalidator/order_validator_test.go

```diff
@@ -435,6 +446,60 @@ func TestComputeOptimalChunkSizesMultiAssetOrder(t *testing.T) {
 	assert.Equal(t, expectedChunkSizes, chunkSizes)
 }
 
+func abiEncode(signedOrder *zeroex.SignedOrder) ([]byte, error) {
+	trimmedOrder := signedOrder.Trim()
+
+	devUtilsABI, err := abi.JSON(strings.NewReader(wrappers.DevUtilsABI))
+	if err != nil {
+		return []byte{}, err
+	}
+
+	data, err := devUtilsABI.Pack(
+		"getOrderRelevantStates",
+		[]wrappers.LibOrderOrder{trimmedOrder},
+		[][]byte{signedOrder.Signature},
+	)
+	if err != nil {
+		return []byte{}, err
+	}
+
+	dataBytes := hexutil.Bytes(data)
+	encodedData, err := json.Marshal(dataBytes)
+	if err != nil {
+		return []byte{}, err
+	}
+
+	return encodedData, nil
+}
+
+func TestComputeABIEncodedSignedOrderStringByteLength(t *testing.T) {
+	signedOrder := scenario.NewSignedTestOrder(t)
+
+	encoded, err := abiEncode(signedOrder)
+	require.NoError(t, err)
+	expectedLength := len(encoded) - emptyGetOrderRelevantStatesCallDataStringLength
+
+	length := computeABIEncodedSignedOrderStringLength(signedOrder)
+
+	assert.Equal(t, expectedLength, length)
+}
+
+func TestMultiAssetComputeABIEncodedSignedOrderStringByteLength(t *testing.T) {
+	signedOrder := scenario.NewSignedTestOrder(t)
+	signedOrder.Order.MakerAssetData = common.Hex2Bytes("123412304102340120350120340123041023401234102341234234523452345234")
+	signedOrder.Order.MakerAssetData = common.Hex2Bytes("132519348523094582039457283452")
+	signedOrder.Order.MakerAssetData = multiAssetAssetData
+	signedOrder.Order.MakerAssetData = common.Hex2Bytes("324857203942034562893723452345246529837")
+
+	encoded, err := abiEncode(signedOrder)
+	require.NoError(t, err)
+	expectedLength := len(encoded) - emptyGetOrderRelevantStatesCallDataStringLength
+
+	length := computeABIEncodedSignedOrderStringLength(signedOrder)
+
+	assert.Equal(t, expectedLength, length)
+}
```

Comment:
Can you combine these into a single test case using table driven testing? An example of this pattern can be found here: https://github.com/0xProject/0x-mesh/blob/master/orderfilter/filter_test.go#L27. 

Let's add one more case that makes use of the `multiAssetData` from the other file.